src/scss/objects/_form-toggle.scss

```diff
@@ -218,11 +216,21 @@ $formToggle-disabled-text           : $grey--lighter;
         }
     }
 
+    .o-formToggle--withCount {
```

Comment:
Might be better to change this to `o-formToggle--hasCount` simply as that's the naming we use for state classes that indicate a similar thing.